keps/sig-network/20191205-kube-proxy-IP-node-binding.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
This is super annoying, but there's a new format for KEPs - they now want a directory and 2 files.

https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template

Reply:
No problem, I'll update it next week 😄

Reply:
done!